src/server/types.ts

```diff
@@ -201,7 +201,11 @@ export interface NextAuthOptions {
  * [Documentation](https://next-auth.js.org/configuration/options#theme) |
  * [Pages](https://next-auth.js.org/configuration/pages)
  */
-export type Theme = "auto" | "dark" | "light"
+export type Theme = {
+  color: "auto" | "dark" | "light"
+  logo?: string
+  brandColor?: string
+}
```

Comment:
```suggestion
export type Theme = {
  colorScheme: "auto" | "dark" | "light"
  logo?: string
  brandColor?: string
}
```

I might want to call this `colorScheme` (after the CSS query "prefers-color-scheme"), or `mode` instead, but not really important to me, I'm fine with `color`.

Do we want to support `"auto" | "dark" | "light"` as well for backward compatibility?

We kind of stopped breaking things since the first beta, but it seems like a reasonably simple thing to upgrade and if we wouldn't do it now, then we would have to wait until v5 to deprecate. But I'm fine with anything we agree on.

Reply:
Yeah good call on this! I didn't like it, but was strugglign to come up with a better name. `colorScheme` is great!